src/plugins/discover/public/application/main/components/sidebar/discover_sidebar_responsive.tsx

```diff
@@ -285,39 +231,11 @@ export function DiscoverSidebarResponsive(props: DiscoverSidebarResponsiveProps)
             fieldCounts={fieldCounts.current}
             setFieldFilter={setFieldFilter}
             editField={editField}
-            createNewDataView={createNewDataView}
           />
         </EuiHideFor>
       )}
       <EuiShowFor sizes={['xs', 's']}>
         <div className="dscSidebar__mobile">
-          <section
-            aria-label={i18n.translate(
-              'discover.fieldChooser.filter.indexAndFieldsSectionAriaLabel',
-              {
-                defaultMessage: 'Index and fields',
-              }
-            )}
-          >
-            <EuiFlexGroup direction="row" gutterSize="s" alignItems="center" responsive={false}>
-              <EuiFlexItem grow={true}>
-                <DiscoverIndexPattern
```

Comment:
So what this did was in mobile view, it added the data view selector additionally inside the flyout with all the fields. It would be really great if we could keep that functionality. Is it possible to have a second data view selector in discover in mobile view?
<img width="412" alt="Screen Shot 2022-03-10 at 18 09 59 PM" src="https://user-images.githubusercontent.com/549577/157770617-c9e96820-ff67-49d3-8114-ebb97c2cffe3.png">

Reply:
Done, I am keeping it only on the flyout on the mobile view